python-backend/app/api/mines/mine/models/mine.py

```diff
@@ -97,17 +96,22 @@ class MineDetail(AuditMixin, Base):
     mine_no = db.Column(db.String(10))
     mine_name = db.Column(db.String(60), nullable=False)
     mine_note = db.Column(db.String(300), default='')
-    major_mine_ind = db.Column(db.Boolean, nullable=False, default=False)
+    major_mine_ind = db.Column(db.Boolean, nullable=False, default=False) 
+    mine_region = db.Column(db.String(2), db.ForeignKey('mine_region_code.mine_region_code'))
+
 
     def __repr__(self):
         return '<MineDetail %r>' % self.mine_guid
 
     def json(self):
+        region_description=MineRegionCode.find_by_region_code(self.mine_region).json()
```

Comment:
I'm not familiar with this part of the codebase, but is there a need to convert it to JSON before pulling out the description on line 114?